qcore/geo.py

```diff
@@ -2,24 +2,27 @@
 Various tools which may be needed in various processes.
 """
 
-from math import sin, asin, cos, atan, atan2, degrees, radians, sqrt, pi
 import os
+import numpy as np
```

Comment:
import builtin1 (os, sys, math, subprocess)
import builtin2

import external1 (numpy, h5py etc..)
import external2

import local1 (qcore or other local files)
import local2

i try to go with the above format, alphabetical within blocks, not sure why they have been rearranged

Reply:
Sounds good, but should we enforce this rule? I think we need to have a set of internal rules that everyone can agree and follow